api/QCVOC.Api/Common/Utility.cs

```diff
@@ -179,6 +179,11 @@ public static T GetSetting<T>(string settingName, T defaultValue)
         /// <returns>The formatted error string.</returns>
         public static string GetReadableString(this ModelStateDictionary dictionary)
         {
+            if (dictionary == null || dictionary.IsValid)
```

Comment:
Alternatively you could use `dictionary?.Keys`.  I think it's good to check IsValid here as well so I think what you have is good.